operator/api/server.go

```diff
@@ -152,16 +160,21 @@ func (s *Server) StartServer() error {
 					log.WithError(err).Error("apiserver shutdown")
 				}
 			case err := <-errCh:
-				log.WithError(err).Warn("Unable to start status api")
+				log.WithError(err).Warn("Unable to start operator API server")
 			}
 		}()
+
 		log.Infof("Starting apiserver on address %s", addr)
 	}
 
 	for err := range errs {
```

Comment:
I think we can move L105 to just above this for-loop so that the variable is closer in scope to its use.